java/core/src/java/org/apache/orc/OrcFile.java

```diff
@@ -691,7 +691,7 @@ public WriterOptions physicalWriter(PhysicalWriter writer) {
     }
 
     /**
-     * A package local option to set the memory manager.
+     * A public option to set the memory manager.
```

Comment:
We usually don't mix like this. Especially, this minor change exists in an independent file. Please revert this. You can do this later in another PR.

Reply:
I raised another PR for this line: https://github.com/apache/orc/pull/923 
I can rebase against the upstream once that's merged in.

Reply:
Why do you want to blocked by another PR, @autumnust ?